include/Dialect/Pulse/Transforms/Scheduling.h

```diff
@@ -45,6 +48,9 @@ struct QuantumCircuitPulseSchedulingPass
   QuantumCircuitPulseSchedulingPass(SchedulingMethod inSchedulingMethod) {
     SCHEDULING_METHOD = inSchedulingMethod;
   }
+  QuantumCircuitPulseSchedulingPass(uint64_t inPreMeasureBufferDelay) {
```

Comment:
Would there be a case where both the `SCHEDULING_METHOD` and `PRE_MEASURE_BUFFER_DELAY` would need to be set at this same time?